src/alire/alire-origins.ads

```diff
@@ -39,10 +39,10 @@ package Alire.Origins with Preelaborate is
      );
 
    Supports_Hashing : constant array (Kinds) of Boolean :=
-                        (Git |
+                        (Filesystem |
+                         Git |
```

Comment:
What do we do with git origin? Only allowed in for local crates?

Reply:
Currently it is using the "obsolete" `git archive` way. We should deprecate that, I guess.